watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -0,0 +1,634 @@
+        is_output=False,
+    )
+
+    # ED stack conditions
+    exports.add(
+        obj=fs.EDstack.voltage_applied[0],
+        name="Stack voltage",
+        ui_units=pyunits.mol / pyunits.m**3,
+        display_units="V",
+        rounding=3,
+        description="Applied constant voltage on the ED stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+    exports.add(
+        obj=fs.EDstack.cell_pair_num,
+        name="Cell pair number",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=0,
+        description="Cell pair number in a single ED stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+    exports.add(
+        obj=fs.EDstack.channel_height,
+        name="Channel height",
+        ui_units=pyunits.meter,
+        display_units="m",
+        rounding=4,
+        description="Channel height of the ED flow channels",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+    exports.add(
+        obj=fs.EDstack.cell_width,
+        name="Cell width",
+        ui_units=pyunits.meter,
+        display_units="m",
+        rounding=3,
+        description="The width of ED cell or stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+    exports.add(
+        obj=fs.EDstack.cell_length,
+        name="Channel length",
+        ui_units=pyunits.meter,
+        display_units="m",
+        rounding=3,
+        description="The length of ED cell or stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_porosity,
+        name="Spacer porosity",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Porosity of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_porosity,
+        name="Spacer porosity",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Porosity of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_specific_area,
+        name="Spacer specific surface area",
+        ui_units=pyunits.meter**-1,
+        display_units="m^-1",
+        rounding=0,
+        description="Specific surface area of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.electrodes_resistance,
+        name="Electrode resistance",
+        ui_units=pyunits.ohm * pyunits.meter**2,
+        display_units="ohm m^2",
+        rounding=2,
+        description="Areal resistance of the two electrodes",
```

Comment:
```suggestion
        description="Area resistance of the two electrodes",
```

Reply:
This is a word mostly used in the literature so I would keep it.